rmf_demo_maps/maps/airport_terminal/airport_terminal.building.yaml

```diff
@@ -161,6 +161,36 @@ levels:
         vertices: [1241, 1227, 1240, 1228, 1239, 1229, 1237, 1230, 1238, 1231, 1232, 1233, 1234, 1235, 1236, 1226]
       - parameters: {texture_name: [1, concrete2], texture_rotation: [3, 0], texture_scale: [3, 1]}
         vertices: [1235, 1234, 1233, 1232, 1231, 1238, 1197, 262, 261]
+    human_lanes:
```

Comment:
You seem to have added human lanes to the `airport_terminal` world but there is no mention in the PR about crowd simulation in this scenario?

Reply:
I used to test with airport crowdsim, when I pull the latest git from osrf/rmf_demos, it keeps the remnants of old content but does not replace the old file entirely. The actual demos only cover office and clinic maps. So, don't merge this airport map.

Reply:
We can't choose which files to merge and which to skip so all the unnecessary changes should be removed on your side